spyder/plugins/projects/widgets/main_widget.py

```diff
@@ -168,8 +169,13 @@ def __init__(self, name, plugin, parent=None):
         self.treewidget.sig_open_file_requested.connect(
             self.sig_open_file_requested)
 
-        # Empty widget to show when there's no active project
-        self.emptywidget = ProjectExplorerTreeWidget(self)
+        self.pane_empty = PaneEmptyWidget(
+            self,
+            "projects",
+            _("You haven't opened a project yet."),
+            _("To create a new project go to the menu entry Projects > "
+              "New project")
```

Comment:
```suggestion
            _("Create one using the menu entry Projects > New project.")
```

Be more direct and concise, and avoid redundancy and duplication